docs/developer-docs/latest/setup-deployment-guides/installation/templates.md

```diff
@@ -76,7 +76,29 @@ To create a Strapi template, you need to publish a public GitHub repository that
 
 First, a template's only concern should be to adapt Strapi to a use case. It should not deal with environment-specific configs, like databases, or upload and email providers. This is to make sure that templates stay maintainable, and to avoid conflicts with other CLI options like `--quickstart`.
 
-Second, a template must follow the following file structure.
+Second, a template must follow the file structure detailed below.
+
+You can create this file structure by hand or generate it via the [CLI](/documentation/developer-docs/latest/developer-resources/cli/CLI.html#strapi-generate-template)
+
+:::: tabs
```

Comment:
Just a FYI (please don't change your PR content for now):

We will switch to more compact/prettier tabs. This is going to be experimented with the upcoming rewritten Quick Start Guide.

We'll use [VuePress code groups](https://vuepress.vuejs.org/theme/default-theme-config.html#code-groups-and-code-blocks) for a end result like this:
![Screenshot 2021-06-04 at 14 56 44](https://user-images.githubusercontent.com/4233866/120804582-1c1cd900-c545-11eb-89a8-ee25bb98339d.png)


Just so you know for upcoming PRs :-) Thank you!